apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -342,6 +390,9 @@ def from_anndata(
                 platform_config=platform_config,
                 context=context,
                 ingestion_params=ingestion_params,
+                # XXX CHECK EXISTENCE OF THE LAYER -- FATAL IF NOT PRESENT
+                # XXX HAVE CHECKED THIS EARLY FOR FAIL-FAST
```

Comment:
Is this something you wanted to address now or is this a TODO for later?

Reply:
@thetorpedodog I am addressing that now & had worked off the to-do and removed the XXX comment.

Let me double-check my work.

Reply:
OK sorry

Just re-pushed